tests/CMakeLists.txt

```diff
@@ -118,6 +118,7 @@ if(ENABLE_PYTHON_SCRIPTING_RESULT)
   add_py_test(test1013.py "NotoSerifTibetan-Regular.ttf" "Read glyph names from unusual post table")
   add_py_test(test1014.py "PfEd layers round tripping")
   add_py_test(test1015.py "Caliban.sfd" "Reverse chaining tables")
+  add_py_test(test1016.py "feta20.pfb" "selection-based isWorthOutputting method")
```

Comment:
I would write as `Tests for font.isWorthOutputting`

Reply:
There is already a glyph-based method as @khaledhosny noted.

Reply:
…so? Unrelated.

Reply:
"Selection-based" vs the existing "glyph-based". I think the current wording is appropriate.

Reply:
I guess that's okay. But the capitalization is definitely inconsistent with that of the other test descriptions.